16/umbraco-cms/customizing/back-office-signs.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Describes how to use sign information provided in management API responses to present additional details to consumers.
+---
+
+# BackOffice Signs
+
+When trees, collections and items are presented in the backoffice, additional information can be displayed in the form of signs. Which signs to present will be determined by the management API response, that will be populated with a collection of `signs`.
```

Comment:
```suggestion
When trees, collections and items are presented in the backoffice, additional information can be displayed in the form of Signs. Which Signs to present will be determined by the Sign Manifest.

A Sign can utilize conditions in the same way as other Extensions or it can be bound to a Flag coming from the Manegement API.
```